docs/tutorials/RemovingFields.md

```diff
@@ -87,12 +87,12 @@ At this point, your application is working as intended and all your tests are pa
 
 In `src/test/data/`, data meant for testing purposes are stored. While keeping the `address` field in the json files does not cause the tests to fail, it is not good practice to let cruft from old features accumulate.
 
-**`invalidPersonAddressBook.json`:**
+**`invalidClientAddressBook.json`:**
 
 ```json
 {
-  "persons": [ {
-    "name": "Person with invalid name field: Ha!ns Mu@ster",
+  "Clients": [ {
```

Comment:
I think this is what wei jie means. Perhaps you can change this jsons from "Clients" to "clients"

Reply:
Or should we ignore the GitHub's checkstyle warning for now and merge, and fix it along the way?

Reply:
hmm yeap I thought so too, I've already edited those json files but the 2 errors in the screenshot above is still present though

Reply:
I think we should merge with the 2 test failure and create a new issue tracker to debug. Then can find out the root cause. 
It seems to happen during the execution of the test (cuz temp file).

Reply:
okay sure